src/commands/UploadReactNativeCommand.ts

```diff
@@ -225,7 +225,11 @@ function validatePlatformOptions(opts: Record<string, unknown>): void {
 }
 
 function validateRetrieval(opts: Record<string, unknown>): void {
-  if (!opts.fetch){
+  if (!opts.fetch && !opts.sourceMap && !opts.bundle) {
+    throw new Error('--fetch or both --source-map and --bundle are required parameters')
```

Comment:
I don't know if describing them as "required" parameters is helpful.
I think might be better off saying something like:

```suggestion
    throw new Error('Not enough arguments provided. Either use --fetch mode, or provide both --source-map and --bundle.')
```
```